src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -9,28 +12,37 @@
 @CrossOrigin("*")
 public class CartController {
 
+    @Autowired
+    private CartService cartService;
+
     @PostMapping(value = "createCart")
     public  CartDto createCart(@RequestParam long userId) {
-        return new CartDto();
+        //return new CartDto();
```

Comment:
Czy te zakomentowane linijki mają jakiś cel w tych endpointach ? Jeżeli nie to raczej bym je usunął.